pandas/algos.pyx

```diff
@@ -104,6 +104,21 @@ cdef _take_2d_int64(ndarray[int64_t, ndim=2] values,
             result[i, j] = values[i, indexer[i, j]]
     return result
 
+cdef _take_2d_uint64(ndarray[uint64_t, ndim=2] values,
```

Comment:
not sure why float64/int64/object are here either, they should also be in algos_take_helper.pxi.in

can you move there (could be in same PR or another ok)